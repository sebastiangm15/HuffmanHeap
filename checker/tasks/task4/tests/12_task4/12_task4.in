9
2 N1
3 N2
5 N3
6 N4
4 N5
7 N6
8 N7
9 N8
10 N9
4
N8
N5
N3
N2
