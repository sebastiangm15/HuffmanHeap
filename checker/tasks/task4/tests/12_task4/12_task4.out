N9N3N4N6N7N5N1N2N8
